submissions/best-of-its-genre-part-2-nanogamer7/influence.md

```diff
@@ -0,0 +1,62 @@
+
+Let's get back to the previous article, what does automation even mean, how can it be defined:
+
+{quote: 'One could loosely define automation as "making stuff that makes stuff"' author: Nano/Myself}
+
+Keep in mind this is only one possible definition of the genre, but it already runs into some problems. Let's take Nanocarbon Alloy as an example, a late game material needed for almost all high tier machines: To automatically produce it we first need resource depots for Iron, Graphite, Ammonium, and Titanium. This by itself doesn't seem too unreasonable, but it is already our first issue: Astroneers only automatic transportation is by auto arms, its equivalent of inserters. However most resources don't spawn right next to each other, so to even get all the solid resources together you'd need extensive amounts of inserter chains. But the much bigger problem are two gasses, Helium and Nitrogen, which both aren't available on the same planet as Iron and Titanium, and the only interplanetary transport is by a manned rocket. So already we need to make stuff by hand, with no way to automate.
+
+{Note to editor: I think the explanation of Nanocarbon Alloy is a bit unnecessary long, I'd maybe shorten it if it doesn't fit the flow}
+
+Most of the reason automation in ASTRONEER isn't possible to an extend like in real automation games, is because it isn't an actual automation game. The auto arm—and with it some other tools to automate—only got added in version `1.13`,[^3] a full year after the original `1.0`release of a pure survival game.
+
+Many (survival) games feature some sort of automation today: Minecraft's hopper, added in version `1.5`in early 2013[^4], is the first example of simple machine to machine transfer I could find, and might have influenced Factorio's inserter too, but ASTRONEER's auto arm, the auto miner, the conveyors in TerraTech, ~~the belts in some Overcooked levels~~ are all reminiscent of similar Factorio mechanics, and all got released after Factorio first got popular. And then there are mods: Basically every tech Minecraft mod since `1.6.4` (although those might also be influenced by the earlier Minecraft mods) features automation.
+
+Still, none of the games above have automation at their core. You might spend way to long building complex machineries, intricate factories, and tightly packed rovers, but none of them progress your gameplay directly. You still need to explore, discover, and fight your way further towards your final goal. There can however be made an argument for SkyFactory, one of my favourite Minecraft mod packs, which circumvents the problem of exploration by having nothing to explore.
+
+### Part 3: Simulation, Idle, and Puzzle Games
```

Comment:
I'm not sure what the goal of this section is. It sounds like you're just naming some small indie games that has some automation aspects. Are you trying to say that since Factorio a new type of automation game is becoming popular even in small indie games?